tests/containers/rootless_podman.pm

```diff
@@ -38,6 +38,16 @@ sub run {
 
     my $podman = $self->containers_factory('podman');
 
+    # add testuser to systemd-journal group to allow non-root
+    # user to access container logs via journald event driver
+    # to avoid flakes in SLE15 SP1 & SLE15 SP2 w/ Podman <=4.0.0
+    if (is_sle('=15-SP1') || is_sle('=15-SP2') || is_leap('=15.1') || is_leap('15.2')) {
```

Comment:
@jlausuch I've added checks for Leap here too, but I'm not sure if we run these tests on Leap, can you please confirm once if this is okay?

Reply:
Why don't we run this condition regardless of the OS (only checking the podman version)?

Reply:
That's because it was reported only for SLE15SP1 and 2 and since we wanted to just add the condition for the specific failing case, I added this.

Reply:
Stil, I would do it version dependant, not OS. Cause the version in each OS can change if there are updates requested within the general support or LTSS time.